main.js

```diff
@@ -0,0 +1,42 @@
+class Color {
+  constructor() {
+    this.hexcode = this.#randomHexCode();
+    this.locked = false;
+    this.id = Color.colorsMadeSoFar + 1;
```

Comment:
if we wanted C1, C2 type IDs we could write `this.id = "C" + Color.colorsMadeSoFar + 1;`